client/src/Components/ProductItem.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+interface Props {
+  idx: number;
+  thumbnail: string;
+  title: string;
+  price: string;
+}
+
+const ProductItemContainer = styled.div`
+  position: relative;
+  height: 300px;
+  width: 230px;
+  transition: all 300ms;
+  &: hover {
```

Comment:
오잉 띄어쓰기 했을 때 오류 없었나요? 
https://www.w3schools.com/cssref/sel_hover.asp

```suggestion
  &:hover {
```

Reply:
저 부분에서 특별한 오류는 없었습니다. 일단 수정해둘게요.